test/signup/signup.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest');
+const app = require('../../app');
+const { findUserByEmail } = require('../../app/services/users');
+
+describe('Tests sign up', () => {
+  const user = {
+    name: 'pepito',
+    last_name: 'Perez',
+    mail: 'pepito@wolox.co',
+    pass: 'PassPepit0'
+  };
+
+  test('Successful transaction', async done => {
+    await request(app)
+      .post('/users')
+      .send(user)
+      .expect(201)
+      .expect(async () => {
+        const mailFound = await findUserByEmail(user.mail);
+        if (!mailFound.length) return done(new Error());
+        return done();
+      });
```

Comment:
Refactor this to something like:
```js
await request(...);

const mailFound = await findUserByEmail(user.mail);

expect(mailFound).toExists();

return done();
```

There is no need to make it like a chaining promise, nor return an error.

Reply:
Algo, you can check the fields to match them to your mock object `user`